backend/pkg/api/db/migrations/0019_add_flat_table.sql

```diff
@@ -0,0 +1,18 @@
+-- +migrate Up
+
+create table instance_fact (
```

Comment:
```suggestion
create table if not exists instance_fact (
```
See `0015_add_package_file.sql` for reference

Reply:
Otherwise the tests will fail: https://github.com/flatcar/nebraska/actions/runs/5417059075/jobs/9847587765?pr=656